packages/react-components/react-color-picker-preview/library/src/components/ColorSlider/useColorSliderStyles.styles.ts

```diff
@@ -54,16 +51,16 @@ const useRootStyles = makeResetStyles({
 
 const useStyles = makeStyles({
   horizontal: {
-    minWidth: '120px',
+    minWidth: '200px',
     // 3x3 grid with the rail and thumb in the center cell [2,2] and the hidden input stretching across all cells
     gridTemplateRows: `1fr var(${thumbSizeVar}) 1fr`,
-    gridTemplateColumns: `1fr calc(100% - var(${thumbSizeVar})) 1fr`,
+    gridTemplateColumns: `1fr calc(106% - var(${thumbSizeVar})) 1fr`,
```

Comment:
Could you please explain the rationale behind using `106%`?

Reply:
It's for consistency with Slider width. But I'll check if thee's a better way to make it